src/common/templates/_jfr-helper.tpl

```diff
@@ -41,8 +41,11 @@ preStop:
     - /bin/sh
     - -c
     - |
+      ## Endpoints are disabled.  Give clients 30 seconds to refresh & remove from the LB, then start disabling background processes and other operations
+      sleep 30;
```

Comment:
@jasonmcintosh @rohitk-h jfr are not shipped in SMP. If pipeline-service take 30sec to shut down, can this thing be added specifically for pipeline-service charts ?

cc: @ankit-harness

Reply:
@arya-harness this needs to be present in all services.
Because we do client grpc load balancing, dns refreshes takes 60s to refresh, so services will still have a terminated pod for about 40s until its refreshed.
All grpc services suffer from this

Reply:
This particular function is only enabled when jfr is there.
Customers in onpremise will not have jfr enabled. If this was a "needed" fix, it should have been there as default in charts since long time.
I would suggest modifying the function- by default, sleep for 1m and then shut down Always. do jfr dumps only when jfr flag is enabled.

This will slow all apps using jfr. @harness-jc 

---
Are there any other alternatives apart from lifecycle hooks which can do this?